Libraries/VLM/MediaProcessing.swift

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2024 Apple Inc.
+
+@preconcurrency import CoreImage.CIFilterBuiltins
+import MLX
+
+private let context = CIContext()
+
+// TODO documentation
+public enum MediaProcessing {
```

Comment:
Needs documentation, but see `PaliGemmaImageProvider` which implements

```
SiglipImageProcessor {
  "do_convert_rgb": null,
  "do_normalize": true,
  "do_rescale": true,
  "do_resize": true,
  "image_mean": [
    0.5,
    0.5,
    0.5
  ],
  "image_processor_type": "SiglipImageProcessor",
  "image_seq_length": 1024,
  "image_std": [
    0.5,
    0.5,
    0.5
  ],
  "processor_class": "PaliGemmaProcessor",
  "resample": 3,
  "rescale_factor": 0.00392156862745098,
  "size": {
    "height": 448,
    "width": 448
  }
}
```

from the python transformers code.